### PR TITLE
little things

### PR DESCRIPTION
**Pull request by wimglenn**
Just a few cleanups and fixes for minor grammar/spelling stuff, you can take it or leave it.  Cheers

### CODE REVIEW

File: parse.py [hunk 1]
```diff
@@ -823,7 +825,7 @@ def _to_group_name(self, field):
             elif '_' in field:
                 group = field.replace('_', '_' * n)
             else:
-                raise KeyError('duplicated group name %r' % (field, ))
+                raise KeyError('duplicated group name %r' % field)
```
Comment (r1chardj0n3s) [inline parse.py]: I do not agree with this change; if there is a chance that field is a tuple for some reason the KeyError message formatting will fail with a bizarro error, or display incorrect information.

Comment (wimglenn) [inline parse.py]: Ah, that's right.  I don't use % formatting often and forgot this weird detail.  Will change it tomorrow.

**Comment by wimglenn** [comment]
@r1chardj0n3s   I've now addressed that issue

**Comment by r1chardj0n3s** [comment]
Thanks! And yeah, I'm old school, still using % formatting :-D

**Pull request merged**